cmd/sts-handlers.go

```diff
@@ -272,22 +269,22 @@ func (sts *stsAPIHandlers) AssumeRoleWithJWT(w http.ResponseWriter, r *http.Requ
 	case clientGrants, webIdentity:
 	default:
 		logger.LogIf(ctx, fmt.Errorf("Unsupported action %s", action))
```

Comment:
Convert this into single 

```
err = fmt.Errorf("Unsupported action %s", action)
writeSTSErrorResponse(w, ErrSTSInvalidParameterValue, err)
```

Removing logging where we are sending client conclusive response..